nemo/collections/asr/models/configs/quartznet_config.py

```diff
@@ -0,0 +1,308 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Any, Callable, List, Optional
+
+from omegaconf import MISSING
+
+from nemo.collections.asr.models.configs import ctc_models_config as ctc_cfg
+from nemo.collections.asr.modules.audio_preprocessing import (
+    AudioToMelSpectrogramPreprocessorConfig,
+    SpectrogramAugmentationConfig,
+)
+from nemo.collections.asr.modules.conv_asr import ConvASRDecoderConfig, ConvASREncoderConfig, JasperEncoderConfig
+from nemo.core.config import modelPT as model_cfg
+
+
+# fmt: off
+def qn_15x5():
```

Comment:
should we start the name with " _" here as we don't expect users to call this themselves?

Reply:
Hmm technically users can override this function which returns a constructor using `encoder_cfg_func` in the constructor. So they may or may not use this function explicitly. They can also import this function to get just the encoder config. 

So I think its better to keep them "public" methods, but to access them youd need to import the actual module since its not provided inside `__init__`